hangupsbot/plugins/restrictedadd.py

```diff
@@ -165,3 +187,17 @@ def removebotadd(bot, event, user_id, *args):
         yield from bot.coro_send_message(
             event.conv,
             _("user id {} is not authorised").format(user_id))
+
+
+
+def addhocheckwhitelist(bot, event, conv_id):
```

Comment:
Can we make this a toggle instead of an add-only?
It also needs a shorter and clearer name, such as `botaddalerts`